pdpipe/core.py

```diff
@@ -173,14 +173,16 @@ class PdPipelineStage(abc.ABC):
         dataframes, which will be used to determine whether this stage should
         be skipped for input dataframes. See pdp.cond for more information on
         specialised Condition objects.
+    label : str, default None
+        A label of this stage.
```

Comment:
not sure I fully understand the label vs desc. Would we like the label to be the string representation then?

Reply:
Nope. Label should be very short, like `'fix_spikes'`, while a description could be something like `"Fixes spikes in all numeric columns by applying moving-window based smoothing function with parameters being dynamically learned, while smoothing, separately for each column."`

Reply:
You really couldn't use the later conveniently for access as your could the first, as in `pipe['fix_spikes']` or for slicing, like `pipe[['fix_spikes', 'drop_corr']]`.